app/src/js/components/Rules/rule.js

```diff
@@ -27,15 +27,31 @@ import Loading from '../LoadingIndicator/loading-indicator';
 import Metadata from '../Table/Metadata';
 import AsyncCommands from '../DropDown/dropdown-async-command';
 import ErrorReport from '../Errors/report';
+import Breadcrumbs from '../Breadcrumbs/Breadcrumbs';
+
+const breadcrumbConfig = [
+  {
+    label: 'Dashboard Home',
+    href: '/'
+  },
+  {
+    label: 'Rules',
+    href: '/Rules'
+  },
+  {
+    label: 'Rule Overview',
+    active: true
+  }
+];
 
 const metaAccessors = [
-  ['RuleName', 'name'],
+  ['Rule Name', 'name'],
+  ['Timestamp', 'timestamp', fullDate],
   ['Workflow', 'workflow'],
```

Comment:
Also just wondering if we should try to link to the workflow like we link to the provider

Reply:
Sorry again. This is a style ticket. There might be an implementation ticket which I would suggest to add this.